src/symbol.rs

```diff
@@ -0,0 +1,89 @@
+use crate::HandleScope;
+use crate::Isolate;
+use crate::Local;
+use crate::String;
+use crate::Symbol;
+use crate::Value;
+
+extern "C" {
+  fn v8__Symbol__New(
+    isolate: *mut Isolate,
+    description: *const String,
+  ) -> *const Symbol;
+  fn v8__Symbol__ForApi(
+    isolate: *mut Isolate,
+    description: *const String,
+  ) -> *const Symbol;
+  fn v8__Symbol__Description(this: *const Symbol) -> *const Value;
+}
+
+macro_rules! well_known {
```

Comment:
I'd suggest to move the definition of this macro down, just above where it is used.

Reply:
I think it is already as low as it can go?

Reply:
If you say so I might have overlooked something.